### PR TITLE
[PCDWEBK-3698] Phone selection error on Chrome and Windows

### PR DESCRIPTION
**Pull request by jbelmont**
Removed references to canShareAudio and canShareVideo and used audioCallCapable and VideoCallCapable. Also fixed test errors with newer changes.

Please review @xdumaine

### CODE REVIEW

**Comment by jbelmont** [comment]
"test": "semistandard addon/**/*.js app/**/*.js blueprints/**/*.js tests/**/*.js && ember test

I removed redundant check of node_modules/.bin/semistandard since in the context of package.json it knows to look in node_modules/.bin directory

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "node_modules/.bin/semistandard addon/**/*.js app/**/*.js blueprints/**/*.js tests/**/*.js && ember test",
```
Comment (xdumaine) [inline package.json]: What's this change for? I like using the local dep instead of global one.

Comment (jbelmont) [inline package.json]: right but in the context of package.json it is doing local dep from my understanding

Comment (jbelmont) [inline package.json]: I believe that if you are outside of package.json than the reference for semistandard would be global but in package.json it first looks in local node_modules/.bin directory

Comment (jbelmont) [inline package.json]: Am I wrong?

Comment (andrejewski) [inline package.json]: You right: https://docs.npmjs.com/cli/run-script

**Pull request merged**